README.md

```
@@ -106,6 +106,13 @@ export const siteConfig: SiteConfig = {
 		},
 	],
 	pinnedPost: ["xxx/yyy.md , zzz.md"], // 置顶文章的 slug 路径，如果为空则不置顶，按顺序排序多个置顶，逗号分隔
+	performance: {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)